src/calendar/day/index.tsx

```diff
@@ -78,15 +77,15 @@ export default class Day extends Component<DayProps> {
   }
 
   getAccessibilityLabel = memoize((day, marking, isToday) => {
-    const today = get(XDate, 'locales[XDate.defaultLocale].today');
-    const formatAccessibilityLabel = get(XDate, 'locales[XDate.defaultLocale].formatAccessibilityLabel');
+    const today = XDate.locales[XDate.defaultLocale].today || 'today';
+    const formatAccessibilityLabel = XDate.locales[XDate.defaultLocale].formatAccessibilityLabel;
     const markingLabel = this.getMarkingLabel(marking);
 
     if (formatAccessibilityLabel) {
       return `${isToday ? today : ''} ${day.toString(formatAccessibilityLabel)} ${markingLabel}`;
     }
 
-    return `${isToday ? 'today' : ''} ${day.toString('dddd d MMMM yyyy')} ${markingLabel}`;
+    return `${isToday ? today : ''} ${day.toString('dddd d MMMM yyyy')} ${markingLabel}`;
```

Comment:
You can loos the condition (lines 84-86) if you pass `const format = formatAccessibilityLabel || 'dddd d MMMM yyyy';` to the day.toString(format) on line 88;

Reply:
Right, I hadn't realised, thanks 😄